pkgs/applications/networking/remote/x2gokdriveclient/default.nix

```
@@ -0,0 +1,66 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  qt5,
+  qtbase,
+  qtx11extras,
+  qttools,
+  zlib,
+  gnumake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "x2gokdriveclient";
+  version = "0.0.0.1";
+
+  src = fetchurl {
+    url = "https://code.x2go.org/releases/source/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "XO6McNZUEQ67TIa9AMHb0qVb5q4kHkZef6GMhpbDdvs=";
+  };
+
+  buildInputs = [
+    qtbase
+    qtx11extras
+    qttools
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-warn "SHELL=/bin/bash" "SHELL=$SHELL" \
+      --replace-warn "MAKEOVERRIDES" "NOMAKEOVERRIDES " \
+      --replace-warn ".MAKEFLAGS" ".NOFLAGS " \
+      --replace-warn "qmake" "${qtbase.dev}/bin/qmake" \
+      --replace-warn "-o root -g root" ""
```

```suggestion
      --replace-fail "SHELL=/bin/bash" "SHELL=$SHELL" \
      --replace-fail "MAKEOVERRIDES" "NOMAKEOVERRIDES " \
      --replace-fail ".MAKEFLAGS" ".NOFLAGS " \
      --replace-fail "qmake" "${qtbase.dev}/bin/qmake" \
      --replace-fail "-o root -g root" ""
```
we want to fail when those no longer match on an update to notice changes early and loud

---

Thanks, I was wondering why the x2goclient package does it like that. Seems way more explicit like this.